packages/db/migrations/sqlite/meta/_journal.json

```diff
@@ -183,6 +183,13 @@
       "when": 1739468826756,
       "tag": "0025_add-group-home-board-settings",
       "breakpoints": true
+    },
+    {
+      "idx": 26,
+      "version": "6",
+      "when": 1739778992076,
+      "tag": "0026_dapper_chameleon",
```

Comment:
When creating migrations, use the ``--name`` parameter to specify a more useful name.
Example: ``pnpm db:sqlite:migration:generate --name add_XXXXXX``